src/api/app/controllers/message_controller.rb

```diff
@@ -17,12 +17,12 @@ def check_project_and_package
   end
 
   def list
-    if @package
-      @messages = @package.messages
+    @messages = if @package
```

Comment:
@bgeuken, Is possible with [MultilineAssignment](http://rubocop.readthedocs.io/en/latest/cops_style/#stylemultilineassignmentlayout)